### DISCUSSION TITLE
Endpoint for user defined assets

### DISCUSSION THREAD

**Discussion by MariusBrill** [grafana/grafana#38464] [Plugins]
**What would you like to be added**:
An endpoint which would point to a folder in the file system which could be used to load user defined assets into a panel. 

**Why is this needed**:
Currently panels need to be signed and published with all assets needed. There are use cases where users would need dynamically added assets such as icons. However, it would be a great benefit if users could dynamically add assets to a dedicated folder in the file system of the grafana server. This way users could directly add assets for a panel to the server instead of rely on external sources such as urls.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
